include/swift/Reflection/ReflectionContext.h

```diff
@@ -623,7 +623,7 @@ class ReflectionContext
 
         // Perform layout
         if (start)
-          TI = TC.getClassInstanceTypeInfo(TR, *start);
+          TI = TC.getClassInstanceTypeInfo(TR, *start, getReader());
```

Comment:
This is why I think a different callback would be nice. In theory it should be possible to look up TypeInfos without a remote process attached; and you could still have a callback to get C type info.